vars/rollingUpgradePipeline.groovy

```diff
@@ -94,22 +100,24 @@ def call(Map pipelineParams) {
                     }
                 }
                 steps {
-                    timeout(time: 10, unit: 'MINUTES') {
-                        script {
-                            wrap([$class: 'BuildUser']) {
-                                dir('scylla-cluster-tests') {
-                                    checkout scm
-                                    ArrayList base_versions_list = params.base_versions.contains('.') ? params.base_versions.split('\\,') : []
-                                    supportedVersions = supportedUpgradeFromVersions(
-                                        base_versions_list,
-                                        pipelineParams.linux_distro,
-                                        params.new_scylla_repo
-                                    )
-                                    (testDuration,
-                                     testRunTimeout,
-                                     runnerTimeout,
-                                     collectLogsTimeout,
-                                     resourceCleanupTimeout) = getJobTimeouts(params, builder.region)
+                    catchError(stageResult: "FAILURE") {
```

Comment:
Addition of the "catchError" wrapper looks as unrelated to the "azure" parts.
So, better to switch this part to a different commit. And First commit which is "fix(vars): add 'azure' to getCloudProviderFromBackned.groovy map" merge in the main one with all the azure code changes.

Reply:
Fixed - moved the change to a separate commit. Also merged the getCloudProviderFromBackend with other azure-specific changes in the first commit